content_resolver/config_manager.py

```
@@ -118,8 +120,11 @@ def _load_config_repo_v2(self, document_id, document, settings):
             config["source"]["repos"][id]["name"] = name
             try:
                 config["source"]["repos"][id]["baseurl"] = repo_data["baseurl"]
-            except KeyError as err:
-                raise ConfigError(f"'{document_id}.yaml' - is invalid. Repo {id} doesn't list baseurl.") from err
+            except KeyError:
+                # FIXME:  No reference for `yaml_file`
+                raise ConfigError("'{file}.yaml' - is invalid. Repo {id} doesn't list baseurl.".format(
+                    file=yml_file,
+                    id=id))
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)